Adds text redaction tags for paperwork.

SomeoneStoleMyNickname opened the pull request:
:cl:
rscadd: Added redacted tags to text formatting. Use adds a set of font boxes to simulate classic stencil redactions.
/:cl:

The tag in action. (Screenshot taken from [this suggestion](https://forums.baystation12.net/threads/persistent-ingame-reports.8742/) I'm working on.
![redacted](https://user-images.githubusercontent.com/3452002/74100349-449dea00-4b2e-11ea-96b8-980342d055c0.png)

Alex6511 commented:
This should have changelogs and probably a screenshot for feedbck purposes. 

Also they will want you to squash before merging.

SomeoneStoleMyNickname committed:
Added a new digital pencode tag. [redacted] generates R E D A C T E D in black font on a black background to give the illuion of redected content in reports.
Therefore a new line to code/_helpers/text.dm was added as well as needed css to the nano/css/shared.css file.
Finally a discription of the pencode was added to code/modules/codex/entries/paperwork.dm.

thasc commented:
:shipit:

Spookerton commented:
Hi. I've updated the PR title and added a changelog generation entry to the OP.
It's unlikely that this is viable ( https://forums.baystation12.net/posts/107525 ) but we'll talk about it.

Shadowtail117 commented on html/changelogs/SomeoneStoleMyNickname-2.yml:
```diff
@@ -0,0 +1,37 @@
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Added a new digital pencode tag. [redacted] creates the string R E D A C E D in black lettes on black background to give the illusion of redacted contend in reports."
```
You made a typo in the changelog.
